docker-entrypoint-initdb.d/001-init.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.13  Distrib 8.0.34, for Linux (x86_64)
+-- MySQL dump 10.13  Distrib 8.0.33, for Linux (x86_64)
 --
 -- Host: localhost    Database: serlo
 -- ------------------------------------------------------
--- Server version	8.0.34
+-- Server version	8.0.33
```

Comment:
Not sure why this changed as I also have
`mysql  Ver 8.0.34 for Linux on x86_64 (Source distribution)`

Reply:
anyways, it is just a comment